Makefile

```diff
@@ -167,11 +167,21 @@ $(TEST_DIR)/issue_70/optimized-grammar/issue_70.go: $(TEST_DIR)/issue_70/issue_7
 	$(BINDIR)/pigeon -nolint -optimize-grammar $< > $@
 
 $(TEST_DIR)/issue_70b/issue_70b.go: $(TEST_DIR)/issue_70b/issue_70b.peg $(BINDIR)/pigeon
-	$(BINDIR)/pigeon -nolint --optimize-grammar $< > $@
+	$(BINDIR)/pigeon -nolint -optimize-grammar -support-left-recursion $< > $@
 
 $(TEST_DIR)/issue_80/issue_80.go: $(TEST_DIR)/issue_80/issue_80.peg $(BINDIR)/pigeon
 	$(BINDIR)/pigeon -nolint $< > $@
 
+$(TEST_DIR)/left_recursion/left_recursion.go: $(TEST_DIR)/left_recursion/left_recursion.peg $(TEST_DIR)/left_recursion/standard/left_recursion.go $(TEST_DIR)/left_recursion/optimized/left_recursion.go $(BINDIR)/pigeon
+	$(BINDIR)/pigeon -nolint -support-left-recursion $< > $@
+
+$(TEST_DIR)/left_recursion/standard/left_recursion.go: $(TEST_DIR)/left_recursion/left_recursion.peg $(BINDIR)/pigeon
+	$(BINDIR)/pigeon -nolint -support-left-recursion $< > $@
```

Comment:
I think this is not necessary, since this parser is the same as the one in `$(TEST_DIR)/left_recursion/`. For the reuse of the tests, see the example of JSON in `examples/json`.

Reply:
Fixed